test/test_library.py

```diff
@@ -1306,6 +1306,28 @@ def test_write_date_field(self):
         item.write()
         assert MediaFile(syspath(item.path)).year == clean_year
 
+    def test_write_multi_genres(self):
+        item = self.add_item_fixture(genre="old genre")
+        item.write(
+            tags={"genres": ["g1", "g2"]},
+        )
+
+        # Ensure it reads all genres
+        self.assertEqual(MediaFile(syspath(item.path)).genres, ["g1", "g2"])
```

Comment:
We have migrated away from `unittest` assertions, so try using `assert ... == ...`